services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/_constarts.py

```diff
@@ -0,0 +1,6 @@
+from typing import Final
+
+_NANO: Final[int] = int(1e9)
+
+SERVICE_RESTART_MAX_ATTEMPTS: Final[int] = 100
```

Comment:
1. typo in module's name ``_constants.py``
2. These consts are very specific to docker's restart policy but if I read those here I am not sure what is this about. Either write long names or something like
```python

# task_template->ContainerSpec ...
DOCKER_CONTAINER_SPEC_RESTART_POLICY_DEFAULTS = {
                "Condition": "on-failure",
                "Delay": 100,
                "MaxAttempts":1*_NANO,
}
```

and then used as

```python
    x =  "task_template": {
            "ContainerSpec": {
                    "RestartPolicy": {**DOCKER_CONTAINER_SPEC_RESTART_POLICY_DEFAULTS, "foo": 33}
```

Reply:
This `DOCKER_CONTAINER_SPEC_RESTART_POLICY_DEFAULTS ` looks better, will us it.